ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -386,6 +386,24 @@
               </v-btn>
             </template>
             <v-list class="actions__more-actions registration-actions">
+              <v-list-item
+                v-if="areExemptionsEnabled"
```

Comment:
Small but I would prefer `isExemptionsEnabled` for consistency.

Reply:
For sure, can update to `isEmemptionEnabled` 👍